src/ui/public/registry/field_formats.js

```diff
@@ -1,97 +1,105 @@
 import _ from 'lodash';
-import { uiRegistry } from 'ui/registry/_registry';
+import chrome from 'ui/chrome';
 import { FieldFormat } from '../../field_formats/field_format';
+import { IndexedArray } from 'ui/indexed_array';
 
-export const RegistryFieldFormatsProvider = uiRegistry({
-  name: 'fieldFormats',
-  index: ['id'],
-  group: ['fieldType'],
+class FieldFormatRegistry extends IndexedArray {
```

Comment:
I think we should move this into `ui/field_formats/field_formats.js`. IMO this isn't really a registry, and that's fine but we might as well keep it next to the rest of the field formats code.

Reply:
that folder doesn't exist so i'll leave it where it is for now